app/controllers/support_interface/users_controller.rb

```diff
@@ -12,13 +12,54 @@ def index
     def show
       @teacher = IdentityApi.get_teacher(uuid)
       @teacher_name =
-        "#{@teacher.fetch(:firstName)} #{@teacher.fetch(:lastName)}"
+        "#{@teacher.fetch("firstName")} #{@teacher.fetch("lastName")}"
+    end
+
+    def link_dqt
```

Comment:
I think this is due to the requirement of the `link-dqt` URL. Normally with a restful controller it handles actions such as list, show, edit, update and delete and this did not fall into any of those categories. It could be edit but the proposed URL has `identity/users/:uuid/link-dqt/trn`

Reply:
@felixclack what would you suggest? I am aware we are using the user resource and the RESTful verbs are finite for what they can do functionality wise. I am open to suggestions please.

Reply:
Will this still use the `support -> identity` namespace or will it now be `support -> dqt` name space?